### PR TITLE
Code Input - remove autoFocus and add proper accessibility props

### PR DESCRIPTION
**Pull request by jordanAtShopify**
Fixes https://github.com/CovidShield/mobile/issues/51
Fixes https://github.com/CovidShield/mobile/issues/52

Removes auto focus (due to accessibility concerns) and adds some additional accessibility props to the CodeInput component.

### CODE REVIEW

File: src/locale/translations/en.json [hunk 1]
```diff
@@ -130,6 +130,7 @@
     "Cancel": "Cancel",
     "FormIntro": "Please enter your 8 digit COVID Shield code.",
     "Action": "Submit code",
+    "InputLabel": "Covid Shield code input",
```
Comment (svinkle) [inline src/locale/translations/en.json:133]: "input" is not required – this is conveyed with the `TextInput` role.

**Comment by svinkle** [comment]
Not sure if it's just my iOS simulator but the keyboard is no longer displaying. 🤔

Screen Shot 2020-05-28 at 10 53 03 AM
Screen Shot 2020-05-28 at 10 53 11 AM

**Comment by kpeatt** [comment]
Is autofocus required for the keyboard to open?

**Comment by svinkle** [comment]
@kpeatt The keyboard should open on `TextInput` tap/interaction. `autofocus` is not required.

**Comment by jordanAtShopify** [comment]
@svinkle Hmm, I have tested this on both iOS and android both seem fine to me. Which device is your simulator?

**Comment by jordanAtShopify** [comment]
@svinkle Not too sure if we can do that at the moment without getting you a developer account @gavrix is that correct?

**Comment by gavrix** [comment]
You don't need developer account for test mode. You can change signing (and bundle id) to anything and should be able to run on connected device

**Comment by janicduplessis** [comment]
You should probably also remove https://github.com/CovidShield/mobile/blob/master/.eslintrc.js#L35.

Are there cases where autofocus on an input is acceptable accessibility wise? Also what about disabling autofocus only if a screen reader is enabled (https://reactnative.dev/docs/accessibilityinfo#isscreenreaderenabled)?

For the keyboard on the simulator I'm not sure how it interacts with VoiceOver but you can try cmd + k to toggle it, I know sometimes it's buggy and you have to turn it off and back on.

image
image

**Comment by jordanAtShopify** [comment]
@svinkle @janicduplessis I added the isScreenReaderEnabled autofocus in https://github.com/CovidShield/mobile/pull/95

I think it needs testing on an actual device though, are we all ok to merge this as is?

**Comment by janicduplessis** [comment]
Sounds good, I think @svinkle had a comment about the InputLabel message though.

**Pull request merged**